e2e/utils.go

```diff
@@ -521,8 +521,8 @@ func writeDataAndCalChecksum(app *v1.Pod, opt *metav1.ListOptions, f *framework.
 	return checkSum, nil
 }
 
-// nolint:gocyclo,gocognit // reduce complexity
-func validatePVCClone(totalCount int, sourcePvcPath, sourceAppPath, clonePvcPath, clonePvcAppPath string, validateEncryption bool, f *framework.Framework) {
+// nolint:gocyclo,gocognit,nestif // reduce complexity
+func validatePVCClone(totalCount int, sourcePvcPath, sourceAppPath, clonePvcPath, clonePvcAppPath, kms string, validateEncryption bool, f *framework.Framework) {
```

Comment:
Is `validateEncryption` still needed now? From a quick glance, it might always have the same value as `kms != ""`?

Reply:
Yes true. We can avoid using of extra variable and rather use kms != "". Thanks @nixpanic for noticing this. Will make required change.